### PR TITLE
Implement gaining and losing reputation

### PR DESCRIPTION
**Pull request by annabellaMalatinszki**
Functions change_reputation_q and change_reputation_a defined in users_logic.py and called in main.py so that reputation changes upon question or answer votes or an accepted question.

### CODE REVIEW

**Review by alextakacs10 — APPROVED** [review]
I use my power to approve this! I understand everything, well done!

File: users_logic.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            amount = 15
+        elif direction == "down":
+            amount = -15
+    elif direction == "up":
+        amount = 10
+    elif direction == "down":
+        amount = -2
+    SQL1 = """SELECT user_name FROM answer WHERE id = %s;"""
+    data1 = (answer_id,)
+    fetch1 = "one"
```
Comment (markKovacs) [inline users_logic.py]: Replace "one" with "cell" because this way you should get back a tuple ((user_name,)) instead of a string (user_name).

**Review by matebalogh11 — APPROVED** [review]
Nice!:)

**Review by markKovacs — APPROVED** [review]
Perfect. Ready to be merged into develop.

**Pull request closed without merging**